File: src/_/06/get_glyph_arity.ts
// SSoT: file:///Users/s0fractal/OMEGA/I/core/get_glyph_arity.md
import { GLYPH_ARITY_LUT } from "@g05";

export function get_glyph_arity(id: number): number {
  return GLYPH_ARITY_LUT[id & 63];
}

File: src/00/sigma_core/src/ontology_gen/06/get_glyph_arity.rs
// SSoT: file:///Users/s0fractal/OMEGA/src/ontology/core/get_glyph_arity.md
#![allow(unused_imports)]
use super::super::L05::*;

pub fn get_glyph_arity(id: u8) -> u8 {
    GLYPH_ARITY_LUT[(id & 63) as usize]
}
